docs/headless-form/index.md

```diff
@@ -0,0 +1,14 @@
+# Headless form integration
+
+Toucan Core form components are designed to work with [ember-headless-form](https://ember-headless-form.pages.dev/).
+
+Ember-headless-form includes integrations with
+
+- native validation
+- changeset validation
+- custom validation
+- yup
```

Comment:
yup?

Reply:
https://github.com/jquense/yup

Perhaps I should change all of these to links?

Reply:
oh, it's an actual thing! 😆 

yeah, I think links is a good idea